doc/index.rst

```diff
@@ -49,4 +50,3 @@ Indices and tables
 
 .. _DNF Plugins Core: https://dnf-plugins-core.readthedocs.org/en/latest/
 .. _DNF Plugins Extras: https://dnf-plugins-extras.readthedocs.org/en/latest/
-
```

Comment:
libhif don't have any documentation for Python. Referencing just hawkey (which stands for Python bindings to libhif) is fine for now - in hawkey documentation should be the reference to libhif github project though.